app/javascript/packages/document-capture/services/upload.tsx

```diff
@@ -45,17 +43,14 @@ export function toFormData(object) {
  *
  * @return {UploadFormEntryError}
  */
-export function toFormEntryError(uploadFieldError) {
+export function toFormEntryError(uploadFieldError: UploadFieldError) {
```

Comment:
Per previous comment:

```suggestion
export function toFormEntryError(uploadFieldError: UploadFieldError): UploadFormEntryError {
```